app/components/HoverPreview/HoverPreviewMention.tsx

```diff
@@ -24,7 +24,7 @@ const HoverPreviewMention = React.forwardRef(function _HoverPreviewMention(
               size={AvatarSize.XLarge}
             />
             <Flex column gap={2} justify="center">
-              <Title>{name}</Title>
+              <Title>{`${name}(${email})`}</Title>
```

Comment:
Lets put it on the second line with a `middot` separator, also `email` is not guaranteed to be here?

Reply:
>Lets put it on the second line with a middot separator

Sure! 

>also email is not guaranteed to be here?

In case the user can't `readEmail` of the mentioned user...